### PR TITLE
[News] 2021-07-08

### PR DESCRIPTION
**Pull request by ybiquitous**
See also:
- https://github.com/sider/runners/blob/0.51.0/CHANGELOG.md
- sider/runners#2214

### CODE REVIEW

File: docs/tools/javascript/tslint.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 hide_title: true
 ---
 
-# TSLint
+# TSLint (unsupported)
 
-> **DEPRECATED**: TSLint has been deprecated and recommends migrating to [TypeScript ESLint](https://github.com/typescript-eslint/typescript-eslint).
-> See [palantir/tslint#4534](https://github.com/palantir/tslint/issues/4534) for details.
-> And, [`tslint-to-eslint-config`](https://www.npmjs.com/package/tslint-to-eslint-config) helps you migrate it.
+> **UNSUPPORTED**: We have dropped the support for TSLint on July x, 2021. Please migrate to [TypeScript ESLint](https://github.com/typescript-eslint/typescript-eslint).
```
Comment (suetsuguk) [inline docs/tools/javascript/tslint.md:9]:
```suggestion
> **UNSUPPORTED**: We have dropped the support for TSLint on July 8, 2021. Please migrate to [TypeScript ESLint](https://github.com/typescript-eslint/typescript-eslint).
```

Have you set a date? Today?

Comment (ybiquitous) [reply docs/tools/javascript/tslint.md]: Good catch! Thanks! 😄

**Comment by ybiquitous** [comment]
Note: We need to keep [`docs/tools/javascript/tslint.md`](https://github.com/sider/sider-docs/blob/ae69d7aed6c0f1a67b8b860bd2f078b007e023b7/docs/tools/javascript/tslint.md), like [`docs/tools/go/golint.m`](https://github.com/sider/sider-docs/blob/ae69d7aed6c0f1a67b8b860bd2f078b007e023b7/docs/tools/go/golint.md) that was previously dropped.

**Review by suetsuguk — APPROVED** [review]
LGTM 👍 Thank you!

**Pull request merged**